src/main/java/baritone/behavior/ElytraBehavior.java

```diff
@@ -410,20 +439,69 @@ public void onTick(TickEvent event) {
 
         if (forceUseFirework || (!firework
                 && sinceFirework > 10
+                && Baritone.settings().elytraUseFireworks.value
```

Comment:
You also have to make sure it doesn't try to take off using fireworks if this setting is disabled (or document the fact that takeoff will always use fireworks).